.gitignore

```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
+
+# new things bla bla bla
```

Comment:
and one more

Reply:
Here is another comment that is a little longer with some autocomplete words. The new version of this is the first app that I’ve ever used to have a good time with the app.

- Item one
- Item two
- Item three

Reply:
Testing